starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,59 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+body {
+	width: 100%;
+	font-family: "open sans", sans-serif;
+}
+
+.header {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	background: url(/Labs/css_recipes_clone/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg);
```

Comment:
to get your image to display, you should always use a relative link from the file you are in: 

`background: url("../images/apple-pie.jpg");`